File: app/src/main/java/com/codenicely/services/feelbazaar/vendorapp/dashboard/view/CourseListView.java
package com.codenicely.services.feelbazaar.vendorapp.dashboard.view;

import com.codenicely.services.feelbazaar.vendorapp.dashboard.data.MyDashboardResponse;
import com.codenicely.services.feelbazaar.vendorapp.helper.Progressive;
import com.codenicely.services.feelbazaar.vendorapp.helper.Toaster;

public interface CourseListView extends Progressive, Toaster {
    void setData(MyDashboardResponse data);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/course_list/view/CourseListView.kt
package org.wikiedufoundation.wikiedudashboard.ui.course_list.view

import org.wikiedufoundation.wikiedudashboard.ui.course_list.data.ExploreCoursesResponse
import org.wikiedufoundation.wikiedudashboard.util.Progressive
import org.wikiedufoundation.wikiedudashboard.util.Toaster

interface CourseListView : Progressive, Toaster {
    fun setData(data: ExploreCoursesResponse)
}
